lib/resque/scheduler/scheduling_extensions.rb

```diff
@@ -149,6 +174,77 @@ def remove_schedule(name)
 
       private
 
+      def supports_lua?
+        redis_master_version >= 2.5
+      end
+
+      def redis_master_version
+        Resque.redis.info['redis_version'].to_f
+      end
+
+      def startup_sha(refresh = false)
```

Comment:
At Shopify we only use non-persisted schedules because it makes more sense. Basically we want to explicitly tell resque-scheduler exactly what should be in its schedule after every deploy so that the shopify commit sha for the schedule matches what's in resque-scheduler's internals